lib/src/components/board/service.dart

```diff
@@ -112,8 +114,23 @@ class BoardService extends BoardServiceBase {
   }
 
   @override
-  Stream<StreamTicksResponse> streamTicks(ServiceCall call, StreamTicksRequest request) {
-    // TODO: implement streamTicks
-    throw UnimplementedError();
+  Stream<StreamTicksResponse> streamTicks(ServiceCall call, StreamTicksRequest request) async* {
+    final board = _fromManager(request.name);
+
+    final ticks = Queue<Tick>();
+    await board.addCallbacks(request.pinNames, ticks);
+
+    try {
+      while (true) {
```

Comment:
Does this ever end? I don't see a break ever... Not sure if it's supposed to end ever but just wondering

Reply:
No its supposed to continue forever until the client cancels it.